src/cdk-experimental/selection/selection.md

```diff
@@ -0,0 +1,63 @@
+The `selection` package handles managing selection state on components
```

Comment:
FYI, I'll do an editing pass on this after it's submitted since it's easier than doing back and forth review of longer copy.